content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,248 @@
+by the localization, and affect the *Accounting* application, so modifications to the default rules,
+or the creation of new rules, should only be done when necessary.
+
+To view all the rules, go to :menuselection:`Payroll --> Configuration --> Rules`. Click on the drop
+down arrow next to the listed Structure (such as **Regular Pay**) to view all the *Rules* for that
+specific **Structure**.
+
+.. image:: configuration/rules.png
+   :align: center
+   :alt: Rules for each salary structure type.
+
+Rule Parameters
+---------------
+
+Salary Rule Parameters affect the various Rules for salary configuration. New Rule Parameters are
+not recommended to create unless specifically needed by the *Accounting* department. To access the
+parameters, go to :menuselection:`Payroll --> Configuration --> Rule Parameters`.
+
+Other Input Types
+-----------------
+
+When creating payslips, it is sometimes necessary to add other entries for specific circumstances,
+like expenses, reimbursements, or deductions. These other inputs can be seen by going to
+:menuselection:`Payroll --> Configuration --> Other Input Types`.
+
+.. image:: configuration/other-input.png
+   :align: center
+   :alt: Other input types for payroll.
+
+A new *Input Type* can be created by clicking the :guilabel:`Create` smart button. Enter the
+description, the code, and which structure type it applies to. Click the :guilabel:`Save` smart
+button when donesmart button to save the changes, or :guilabel:`Discard` to delete the entry.
+
+.. image:: configuration/input-type-new.png
+   :align: center
+   :alt: Create a new Input Type.
+
+Salary Package Configurator
+===========================
+
+Advantages
+----------
+
+When offering potential employees a position, there can be certain **advantages** to make the offer
+more appealing. Any specific advantage is listed in the advantages section of the configuration
+menu. To see the advantages, go to  :menuselection:`Payroll --> Configuration --> Advantages`.
+
+.. image:: configuration/advantages.png
+   :align: center
+   :alt: Settings available for payroll.
+
+To make a new advantage, click the :guilabel:`Create` smart button. A pop up appears for the
+advantage. Enter the information in the fields, then click the :guilabel:`Save` smart button to save
+the changes, or :guilabel:`Discard` to delete the entry.
+
+.. image:: configuration/new-advantage.png
+   :align: center
+   :alt: List of advantages employee's can have.
+
+
+Personal Info
+-------------
+
+The *Personal Information* section lists all of the fields that are available to enter on the
+employee's card.
+
+.. image:: configuration/personal-info.png
+   :align: center
+   :alt: Personal information that appear on employee cards to enter.
+
+To edit an entry, click on the line, and a box with the configuration for that piece of information
+will appear. To edit the information, click the :guilabel:`Edit` smart button, and modify the entry.
```

Comment:
I'm not sure that the configuration form "appears" so much as clicking on a line just takes you to the form page, if you get what I mean? :)
```suggestion
To edit an entry, click on the line. Then, click the :guilabel:`Edit` button, and modify the entry.
```